### ISSUE TITLE
`cf v3-set-health-check` command also sets `command` even when not specified

### ISSUE THREAD

**Issue by selzoc** [cloudfoundry/cli#1772]
**Please fill out the issue checklist below and provide ALL the requested information.**

- [x] I reviewed open and closed github issues that may be related to my problem.
- [x] I tried updating to the latest version of the CF CLI to see if it fixed my problem.
- [x] I attempted to run the command with `CF_TRACE=1` to help debug the issue.
- [x] I am reporting a bug that others will be able to reproduce.
- [ ] If this is an issue for the v7 beta release, I've read through the [official docs](https://docs.cloudfoundry.org/cf-cli/v7.html) and the [release notes](https://github.com/cloudfoundry/cli/releases).

**Describe the bug and the command you saw an issue with**
Setting a healthcheck should not modify the `command` field on a process.  Once the `command` is set via an API call, it can't be modified by buildpacks in future pushes.

In particular, there is a user that wants to set the `invocation-timeout` but does not want the `command` to be set so that they can pick up new commands out of newer java buildpack versions.

**What happened**
`command` is set to the detected start command

**Expected behavior**
`command` remains `null`

**To Reproduce**
Steps to reproduce the behavior; include the exact CLI commands and verbose output:
1. Run `cf push app-name`
1. Run `cf curl /v2/apps/$(cf app app-name --guid) | jq .entity.command` This is null. 
1. Now update the health-check with `cf v3-set-healthcheck app-name port`
1. Run `cf curl /v2/apps/$(cf app app-name --guid) | jq .entity.command` This is not null, it seems to be set to the detected start command. 

**Provide more context**
- platform and shell details: Mac OS X 10.14 iTerm
- version of the CLI you are running: 6.46.1+4934877ec.2019-08-23
- version of the CC API Release you are on: 2.140.0

Note: As of January 2019, we no longer support API versions older than CF Release v284/CF Deployment v1.7.0 (CAPI Release: 1.46.0 (APIs 2.100.0 and 3.35.0).

Note: In order to complete the v7 beta cf CLI in a timely matter, we develop and test against the latest CAPI release candidate. When v7 cf CLI is generally available, we will start supporting official CC API releases again.

**Comment by abbyachau** [comment]
Hey @selzoc many thanks for this. Do you know if this is an issue on v7 beta? Our plan, since all these prefixed commands are experimental, is to update the v7 command and ask users to use v7 beta instead for v3 commands we've already converted over.

**Comment by selzoc** [comment]
No, it does not happen with the cf7 beta, at least version `7.0.0-beta.24+71d763254.2019-08-20`
